packages/lib-react-components/src/MovableModal/MovableModal.spec.js

```diff
@@ -0,0 +1,106 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import { Rnd } from 'react-rnd'
+import sinon from 'sinon'
+import { MovableModal } from './MovableModal'
+import { Modal } from '../Modal'
+
+const title = 'Modal Heading'
+
+const content = (
+  <p>
+    Leo mollis dictum id dis maecenas consectetur metus elementum vivamus nisl, suscipit tristique lectus nulla mus etiam nisi facilisis magnis, scelerisque ligula montes luctus cursus nibh vulputate parturient risus.
+  </p>
+)
+
+describe('MovableModal', function () {
+  it('should render without crashing', function () {
+    const wrapper = shallow(
+      <MovableModal closeFn={() => { }}>
+        {content}
+      </MovableModal>
+    )
+    expect(wrapper).to.be.ok()
+  })
+
+  describe('with Rnd wrapper', function () {
+    it('should be wrapped by an Rnd component', function () {
+      const wrapper = shallow(
+        <MovableModal closeFn={() => { }}>
+          {content}
+        </MovableModal>
+      )
+      expect(wrapper.find(Rnd)).to.have.lengthOf(1)
+    })
+
+    it('should set the min-height, min-width, and position by props', function () {
+      const minWidth = 800
+      const minHeight = 400
+      const position = { x: 100, y: 100 }
+      const wrapper = shallow(
+        <MovableModal closeFn={() => { }}>
```

Comment:
Is it necessary to pass in this `closeFn` prop if the default prop is the same?

Reply:
Nope. It's a remnant of a copy/paste.